src/main/java/de/muenchen/isi/domain/service/etlInterface/EtlInterfaceService.java

```diff
@@ -14,6 +17,44 @@ public class EtlInterfaceService {
 
     private final EtlInterfaceRepository etlInterfaceRepository;
 
+    /**
+     * Ruft die Reporting-EAI Schnittstelle auf, damit ein ETL-Job zur Übertragung einer Infrastruktureinrichtung
+     * vom Backend zur Reporting DB aufgerufen wird
+     *
+     * @param id ID derInfrastruktureinrichtung
+     * @throws ReportingException falls der Aufruf fehlgeschlagen ist.
+     */
+    public void etlInterfaceTriggerInfrastruktureinrichtungJob(final UUID id) throws ReportingException {
+        final EtlTriggerJobDto etlTriggerJobDto =
+            this.prepareJob("importFromBackend/infrastruktureinrichtung/Job_Import_Infrastruktureinrichtung.kjb", id);
```

Comment:
Wäre es evtl. besser den Jobnamen als Property vorzuhalten?

Reply:
Ich wollte bewusst den Aspekt des Pfadnamens in der jeweiligen Methode (etlInterfaceTrigger_InfrastruktureinrichtungJob_, etlInterfaceTrigger_BauvorhabenJob_) kapseln. Ich würde das gerne so lassen, außer es spricht was dagegen.